src/forwarder.spec.ts

```diff
@@ -20,11 +20,17 @@ function serviceWithResources(resources?: Record<string, any>, serviceName = "my
   return service as Service;
 }
 
-function awsMock(existingSubs: { [key: string]: any }, stackName?: string): Aws {
+function awsMock(existingSubs: { [key: string]: any }, stackName?: string, doesAlwaysReject?: boolean): Aws {
   return {
     getStage: () => "dev",
     request: (service, method, params: any) => {
+      if (doesAlwaysReject) {
+        return Promise.reject("Not found.");
+      }
       const logGroupName = params.logGroupName;
+      if (method == "getFunction") {
+        return Promise.resolve();
+      }
       if (existingSubs[logGroupName]) {
```

Comment:
It looks like this logic here around existing subs is intended for one specific method (getting subscription filters?), I would wrap it in an `if` statement so that it's scoped to that method only.